documentation/commands/git-new-pull-request.md

```diff
@@ -12,5 +12,5 @@ git new-pull-request
 
 #### DESCRIPTION
 
-Opens a browser to a new pull request for the current branch.
+Opens a browser window to the "new pull request" page for the current branch on the code hosting platform.
```

Comment:
What you do think about 
`to the form for a new pull request` 
instead of 
`to the "new pull request" page`